Add "Get Help" page

Tabrarian opened the pull request:
Adding content outline and basic info to create a "Get Help" page on the 2.0 site.

mirons-google commented:
@Tabrarian do you mind combining your planned changes for the /community page to this PR as well? That we we can look at them together and de-dup and ensure the content is consistent.

Tabrarian commented:
Following on yesterday's discussion, I created a couple includes, updated the community page, and added a help page for use and develop. I'm unsure if I got the include syntax correct. PTAL, thanks!

mirons-google commented on _includes/docs/latest/get_help_bug.md:
```diff
@@ -0,0 +1,19 @@
+If you have an idea for a Forseti feature that isn't already on the
+[forseti-security](https://github.com/GoogleCloudPlatform/forseti-security/issues)
+issue list, we'd like to hear from you! Please file a new feature request to the
+issue list so community members and Google engineers can review it.
```
"... so community members and forseti security engineers can review it."

mirons-google commented on _docs/_latest/use/get-help.md:
```diff
@@ -0,0 +1,13 @@
+# {{ page.title }}
+
+This page includes resources if you need help while following the
+[guides]({% link _docs/latest/use/index.md %} to use Forseti.
```
Missing your closing parens after the %}

Tabrarian commented:
I think I got it, thanks, just waiting on Travis!
